zenodb_test.go

```diff
@@ -60,100 +63,6 @@ func TestSingleDB(t *testing.T) {
 	})
 }
 
-func TestClusterPushdownSinglePartition(t *testing.T) {
```

Comment:
This cluster stuff never actually tested the full clustering path because it bypassed RPC. The new tests in server_test.go cover real clusters, so I removed these.